src/main/msp/msp.c

```diff
@@ -1371,6 +1371,7 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, (uint16_t)constrain(gpsSol.llh.altCm / 100, 0, UINT16_MAX)); // alt changed from 1m to 0.01m per lsb since MSP API 1.39 by RTH. To maintain backwards compatibility compensate to 1m per lsb in MSP again.
         sbufWriteU16(dst, gpsSol.groundSpeed);
         sbufWriteU16(dst, gpsSol.groundCourse);
+        sbufWriteU16(dst, gpsSol.hdop);    
```

Comment:
We usally add a comment about the MSP version that supports it, in this case 1.44.

Similar to this:
https://github.com/betaflight/betaflight/blob/a752d3d218a959aa36d51b9d720c75fa95bd62ca/src/main/msp/msp.c#L1359-L1363

Can you add a `// Added in API version 1.44` comment to your change?